HW3/models/platform.js

```diff
@@ -0,0 +1,43 @@
+const chalk = require("chalk");
+module.exports = class platform {
+  constructor(name, users = [], workers = [], bookings = [], id) {
+    this.name = name;
+    this.users = users;
+    this.workers = workers;
+    this.bookings = this.bookings;
+    this.id = id;
+  }
+  registerUser(user) {
+    this.users.push(user);
+  }
+  registerWorker(worker) {
+    this.workers.push(worker);
+  }
+  createBooking(client, worker) {
+    const foundClient = this.users.find((c) => client.email == c.email);
+    const foundWorker = this.workers.find((w) => worker.email == w.email);
+
+    if (foundClient && foundWorker) {
+      const booking = new Booking(this, client, worker);
```

Comment:
You didn't require Booking here in this file, so this wouldn't work. You would notice if you simulated a booking in index.js file for HW3. It makes sense to do so.